### ISSUE TITLE
Recipe ignored when to run multi recipes with module

### ISSUE THREAD

**Issue by linux-china** [casey/just#1823]
root justfile with following code:

```
mod demo

hello:
   echo "root"

hello2:
   echo "root2"
```

module demo/justfile with following code:

```
hello:
   echo "demo"
```

Run multi recipes: `just --unstable demo::hello hello`

but `hello` recipe ignored.

**Comment by casey** [comment]
Nice catch! I'm sure this is a bug, I'll check it out.

**Comment by casey** [comment]
Fixed in #1826, and released in 1.22.1!
